torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -1274,6 +1316,12 @@ TensorView* reductionOp(
   }
   std::sort(uint_axes.begin(), uint_axes.end());
 
+  auto maybe_full =
```

Comment:
Please leave a comment about this behavior, i.e., in PyTorch size-0 reduction effectively means a full op.

Reply:
added